README.md

```diff
@@ -226,7 +230,8 @@ Flask Ask used to have a bug that would not resume the song from the correct off
 Issues where Alexa responds to your requests but doesn't play music are
 generally caused by the `APP_URL` environment variable being set improperly. Be
 sure that it is set to something like `APP_URL=https://ff9b5cce.ngrok.io` 
-**without a trailing slash or `/alexa`**.
```

Comment:
I have no idea how that got thrown in. Must have been an accident. Fixing right now.